dm/master/server.go

```diff
@@ -1230,7 +1230,7 @@ func enableTLS(tlsCfg *config.Security) bool {
 		return false
 	}
 
-	if len(tlsCfg.SSLCA) == 0 || len(tlsCfg.SSLCert) == 0 || len(tlsCfg.SSLKey) == 0 {
+	if len(tlsCfg.SSLCA) == 0 {
```

Comment:
The issue  in https://github.com/pingcap/dm/issues/1555 was a MySQL client side tls issue , but this line of change is DM server side tls config that will effect on the `AdvertiseAddr`.  IIUC, it's unrelated.

Reply:
https://github.com/pingcap/dm/pull/1560/files#diff-d42d00fe16fdbc10836731179db540c30ef59f1eb7789bf2ad3ea771554eceb5R67
The MySQL related file is in pkg/conn/basedb.go. I change this line because it's wrong for dm-master too.

Reply:
Does dm-worker need similar change?

Reply:
I just searched `len(.*SSLCA)`, looks that's the only place left.